src/opera_disp_tms/create_tile_map.py

```diff
@@ -87,17 +89,63 @@ def create_tile_map(output_folder: str, input_rasters: list[str], scale_range: l
         create_bounds_file(vrt_info, scale_range, Path(output_folder))
 
 
-def main():
+def download_geotiffs(bucket, bucket_prefix):
+    resp = list_files_in_s3(bucket, bucket_prefix)
+
+    geotiff_s3_filenames = [f['Key'] for f in resp if f['Key'].endswith('.tif')]
+    dest_dir = Path.cwd()
+
+    geotiff_paths = [
+        download_file_from_s3(bucket, geotiff_s3_filename, dest_dir) for geotiff_s3_filename in geotiff_s3_filenames
+    ]
+
+    return geotiff_paths
```

Comment:
Looks like `download_geotiffs` will return `list[Path]` because `download_file_from_s3` returns `Path`, but then the list gets passed to `generate_tile_map_service` which expects `list[str]`, then to `create_tile_map`, and finally to `gdal.BuildVRT`. Not sure if this will cause a runtime error or if `gdal.BuildVRT` can accept `Path` or `str`. But, it might be a good idea to change the return type of `download_geotiffs` and add type annotations to the function signature to make it more clear what type is being returned.
```suggestion
def download_geotiffs(bucket: str, bucket_prefix: str) -> list[str]:
    resp = list_files_in_s3(bucket, bucket_prefix)

    geotiff_s3_filenames = [f['Key'] for f in resp if f['Key'].endswith('.tif')]
    dest_dir = Path.cwd()

    geotiff_paths = [
        str(download_file_from_s3(bucket, geotiff_s3_filename, dest_dir)) for geotiff_s3_filename in geotiff_s3_filenames
    ]

    return geotiff_paths
```

Reply:
Didn't have any errors when running locally. I think I would rather keep them as Path's since they're more descriptive of what they are and update the types to be paths. Could do a string conversion in the create_tile_map function.